packages/telemed-ehr/app/src/telemed/components/CancelVisitDialog.tsx

```diff
@@ -31,13 +33,16 @@ const CancelVisitDialog = ({ onClose }: CancelVisitDialogProps): ReactElement =>
   const { id: appointmentID } = useParams();
   const navigate = useNavigate();
 
-  const cancellationReasons = [
+  const telemedCancellationReasons = [
     'Patient did not answer after multiple attempts',
     'Wrong patient name on chart',
     'Technical issues connecting and/ or with video',
     'Other',
   ];
 
+  const cancellationReasons =
+    appointmentType === 'telemed' ? telemedCancellationReasons : Object.values(CancellationReasonOptions);
```

Comment:
why does telemed reasons come from this file and in-person reasons come from another file, and why are the variable names not consistent?

Reply:
changed CancellationReasonOptions to inPersonCancellationReasons and moved both to the types file